src/main/java/org/jitsi/jigasi/CallManager.java

```diff
@@ -623,4 +690,60 @@ public static synchronized void restartPool()
 
         threadPool = Executors.newFixedThreadPool(POOL_SIZE);
     }
+
+    /**
+     * A thread factory that creates daemon threads.
+     *
+     * @author Pawel Domas
+     */
+    private static class DaemonThreadFactory
+        implements ThreadFactory
+    {
+        private final String threadName;
+
+        public DaemonThreadFactory(String threadName)
+        {
+            this.threadName = threadName;
+        }
+
+        /**
+         * {@inheritDoc}
+         */
+        @Override
+        public Thread newThread(Runnable r)
+        {
+            Thread t = Executors.defaultThreadFactory().newThread(r);
+            if (!t.isDaemon())
+            {
+                t.setDaemon(true);
+
+                if (threadName != null)
+                {
+                    t.setName(threadName + "-" + t.getId());
+                }
+            }
+            return t;
+        }
+    }
+
+    private static class DummyCallThread
```

Comment:
This is not used anywhere?

Reply:
Yes. Just give a name of the threads for the stack traces.

Reply:
Oh, this is something I forgot to delete :) I was filling pools with this one while testing :P

Reply:
It is deleted now.